app/services/Utils.ts

```diff
@@ -513,6 +513,7 @@ const checkFileExistence = async (filePath:string) => {
 //child data get
 export const getChild = async (child: any, genders: any): Promise<any> => {
     try {
+        const autoChild: any = child.autoChild == "true" ? "true" : "false"; 

```

Comment:
Use the following code if you intend to store string values:
`const autoChild: string = child.autoChild === "true" ? "true" : "false";`

Consider the following version if you intend to store boolean values:
`const autoChild: boolean = child.autoChild === "true";`